cfpb/cfgov-refresh
# Remove external overview link field from Wagtail menu item

The `external link` field for overview pages in the Wagtail menu item model was only being used to supply a default value for non-existent overview pages. To reduce the possibility of confusion for users, this removes that field and adds the default value in the mega menu template. (More info in GHE 2816)


## Additions

- Default value (`#`) for `nav_overview_url` parameter in mega menu template
- `Diversity & Inclusion` link in `migrate_mega_menu` script 

## Removals

- External link field in menu item model

## Testing

1. Check out branch
2. Verify that overview links display correctly in the hard-coded version of the mega menu
3. Verify that overview links display correctly in the Wagtail version of the mega menu:  
    - Run `python cfgov/manage.py migrate` and `python cfgov/manage.py runscript migrate_mega_menu`  
    - Turn on `WAGTAIL_MENU` flag
    - Check overview links

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
